templates/components/login-no-pass-form.jsx

```diff
@@ -18,8 +18,8 @@ var LoginNoPassword = React.createClass({
   render: function() {
     return (
       <div className="loginNoPass innerForm centerDiv">
-        <Form origin="Migration" ref="userform" fields={fields} validators={fieldsValidators} />
-        <button onClick={this.processFormData} className="btn btn-awsm">Log in</button>
+        <Form origin="Migration" ref="userform" fields={fields} validators={fieldsValidators} defaultUsername={this.props.username} />
```

Comment:
Just a reminder for you and me that we'll need `IconText` component here with the text @secretrobotron gave us yesterday.